pkgs/top-level/all-packages.nix

```diff
@@ -19399,6 +19399,18 @@ in
 
   gimpPlugins = recurseIntoAttrs (callPackage ../applications/graphics/gimp/plugins {});
 
+  glimpse = callPackage ../applications/graphics/glimpse {
+    gegl = gegl_0_4;
+    lcms = lcms2;
+    inherit (darwin.apple_sdk.frameworks) AppKit Cocoa;
+  };
+
+  glimpse-with-plugins = callPackage ../applications/graphics/glimpse/wrapper.nix {
+    plugins = null; # All packaged plugins enabled, if not explicit plugin list supplied
```

Comment:
```suggestion
    plugins = null; # All packaged plugins enabled, if no list supplied
```

Reply:
this is also an issue with the documentation for gimp. should i add another commit to fix it for them as well?

Reply:
Ah, of course. Yes, though it should go in another PR I think.